07week/dom-practice/js/script.js

```diff
@@ -1,5 +1,30 @@
-'use strict';
-
-document.addEventListener("DOMContentLoaded", function(event) {
-  // You code here
-});
+'use strict';
```

Comment:
Deleting an item from the cart was optional, but there should have been a button to clear the cart